packages/blade/src/components/Indicator/Indicator.tsx

```diff
@@ -65,16 +65,15 @@ const Indicator = ({
   const { theme } = useTheme();
 
   const fillColor = theme.colors.feedback.background[intent].highContrast;
-  const strokeColor = theme.colors.brand.gray[200].lowContrast;
+  const strokeColor = theme.colors.brand.gray.a100.highContrast;
   const getDimension = useCallback((): Dimensions => {
-    // the actual svg size would get reduced by 2 because of stroke width
     switch (size) {
       case 'small':
+        return { svgSize: '6', textSize: 'small' };
```

Comment:
nit: Can we store this in a indicatorTokens file so that all the size mappings are easy to access & change when needed?

Reply:
Can do, though seems like an overkill right now because we only need these two properties 🤔